src/pow/pow.cpp

```diff
@@ -31,15 +31,19 @@ uint32_t GetNextWorkRequired(const CBlockIndex *pindexPrev,
         return pindexPrev->nBits;
     }
 
-    if (IsAxionEnabled(params, pindexPrev)) {
-        return GetNextASERTWorkRequired(pindexPrev, pblock, params);
+    // ASERT requires an anchor block which has one parent.
+    // The Genesis block has no parent. Therefore, the block after Genesis
+    // will be our anchor block.
+    // We assign it minimum difficulty.
+    if (!pindexPrev->pprev) {
+        const uint32_t nProofOfWorkLimit =
+            UintToArith256(params.powLimit).GetCompact();
+        return nProofOfWorkLimit;
```

Comment:
Can you add a TODO or a task in this repo to delete the EDA code?

Reply:
In fact, the EDA, DAA and Grasberg code. Grasberg makes 0 sense for our coin.